Backend/Backend/Controllers/PlaylistController.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+using Backend.Controllers.Requests;
+using Backend.Services;
+using Backend.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backend.Controllers;
+
+[ApiController]
+[Route("playlist")]
+[Authorize]
+public class PlaylistController : ControllerBase
+{
+    private readonly PlaylistService _service;
+    private readonly IAuthService _authService;
+
+    public PlaylistController(PlaylistService service, IAuthService authService)
+    {
+        _service = service;
+        _authService = authService;
+    }
+    
+    [HttpPost("create")]
+    public async Task<IActionResult> Create([FromBody] PlaylistCreateRequest request)
+    {
+        return Ok(await _service.Create(
+                await _authService.IdentifyUserAsync(HttpContext)!, request.Name));
+    }
+    
+    [HttpPut("append")]
+    public async Task<IActionResult> Append([FromBody] PlaylistAppendRequest request)
+    {
+        bool result = await _service.Append(
+            await _authService.IdentifyUserAsync(HttpContext)!, request.PlaylistId, request.EpisodeId);
+
+        return result ? Ok() : BadRequest("Invalid Playlist ID or Episode ID");
+    }
+
+    [HttpGet("all")]
```

Comment:
all what? might want to add comments to these. Is it all episodes in the podcast? or all episodes selected in the playlist. kinda confusing